mmv1/third_party/terraform/services/resourcemanager/resource_google_project_service.go.erb

```diff
@@ -271,7 +287,11 @@ func resourceGoogleProjectServiceUpdate(d *schema.ResourceData, meta interface{}
 }
 
 // Disables a project service.
+<% if version == "ga" -%>
 func disableServiceUsageProjectService(service, project string, d *schema.ResourceData, config *transport_tpg.Config, disableDependentServices bool) error {
+<% else -%>
+func disableServiceUsageProjectService(service, project string, d *schema.ResourceData, config *transport_tpg.Config, disableDependentServices bool, checkUsage bool) error {
+<% end -%>
```

Comment:
can these be inverted to be an unless/else?

Reply:
Done